src/ngraph/opsets/opset3_tbl.hpp

```diff
@@ -22,4 +22,6 @@
 #include "opset2_tbl.hpp"
 NGRAPH_OP(NonZero, ngraph::op::v3)
 NGRAPH_OP(ROIAlign, ngraph::op::v3)
+NGRAPH_OP(ScatterAdd, ngraph::op::v3)
```

Comment:
Please remove this operation from opset3 table. There is no such operation in opset3 documentation yet.
```suggestion
```